flags.go

```diff
@@ -239,39 +230,36 @@ func newApp() (app *cli.App) {
 			cli.StringFlag{
 				Name:  "temp-dir",
 				Value: "",
-				Usage: "Absolute path to temporary directory for local GCS object " +
-					"copies. (default: system default, likely /tmp)",
+				Usage: "Path to the temporary directory where writes are staged prior to" +
+					" upload to Cloud Storage. (default: system default, likely /tmp)",
 			},
 
 			cli.StringFlag{
 				Name:  "client-protocol",
 				Value: string(mountpkg.HTTP1),
-				Usage: "The protocol used for communicating with GCS backend. " +
+				Usage: "The protocol used for communicating with the GCS backend. " +
 					"Value can be 'http1' (HTTP/1.1) or 'http2' (HTTP/2).",
 			},
 
 			cli.IntFlag{
 				Name:  "max-conns-per-host",
 				Value: 100,
-				Usage: "The max number of TCP connections allowed per server. " +
-					"This is effective when --client-protocol is set as 'http1'.",
+				Usage: "The max number of TCP connections allowed per server. This is " +
```

Comment:
correction: remove space after full stop at the end.

Reply:
fixed